src/cgame/cg_local.h

```diff
@@ -649,6 +649,9 @@ typedef struct clientInfo_s
 	int breathPuffTime;
 	int cls;
 	int latchedcls;
+	int spawnpt;
+	int mspawnpt;
```

Comment:
I do wonder if this should be renamed to something else (and also the key that is written to client info string), `mspawnpt` can be interpreted as either major or minor spawnpoint :D This would be easy to just change to `int minorspawnpt`, but I don't really have a suggestion for the client info string, maybe just `misp`, dunno. But this isn't required really, it's not very hard to figure out what this is for someone who's gonna look at this in few months.